### DISCUSSION TITLE
[Feature Request]: 识别词的范围希望能够从文件级别提升到目录级别，部分合集的刮削信息并不在于视频文件的命名上

### DISCUSSION THREAD

**Discussion by ColaHikari** [jxxghp/MoviePilot#3478] [讨论]
### 当前程序版本

1.9.0

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

![image](https://github.com/jxxghp/MoviePilot/assets/123285105/299a6caa-6045-4f65-8c58-5199fec0264b)
如上图，对于这个合集，刮削信息在目录上，导致刮削的时候永远识别为s1，希望能在识别词获取信息的范围上扩充到目录上，从而更有效的替换实际的视频文件名

**Comment by dahuzi-gege** [comment]
我也想说啊强烈建议啊。。经常替换名字 但是目录名字就是正确的，里面的剧集名字是乱的

**Comment by jxxghp** [comment]
@dahuzi-gege 现在会识别目录的名称啊，举个例子，什么场景不会？
